tests/ghbug335-fail.phpt

```diff
@@ -0,0 +1,34 @@
+--TEST--
+GH Bug #335: Test that APCu stampede protection can be disabled
+--SKIPIF--
+<?php require_once(dirname(__FILE__) . '/skipif.inc'); ?>
+if (!extension_loaded('pcntl') die('skip pcntl required');
```

Comment:
This line should be part of the `<?php ?>` block :)

Reply:
Uhhhh, indeed ...